src/test/java/com/google/api/codegen/testdata/python_doc_main_library.baseline

```diff
@@ -0,0 +1,876 @@
+============== file: library/library_service_api.py ==============
```

Comment:
Ah, looks like the reason it's not showing up in the diff is that git treats the new file as a renaming from the old baseline test for the non-doc test.